scripts/api-inference/pnpm-lock.yaml

```diff
@@ -9,8 +9,8 @@ importers:
   .:
     dependencies:
       '@huggingface/tasks':
-        specifier: ^0.11.11
-        version: 0.11.11
+        specifier: ^0.12.15
```

Comment:
Latest is `0.13.2` (https://www.npmjs.com/package/@huggingface/tasks) and should fix the import issue. (can be done in a later PR, typically the automated one)

Reply:
indeed! i will add that in https://github.com/huggingface/hub-docs/pull/1480